README.md

```diff
@@ -10,58 +10,59 @@ Then was updated by bovirus and this is the [reworked thread on XDA](https://for
 ![APK-Info](https://github.com/Enyby/APK-Info/blob/master/screenshot.png?raw=true "APK-Info")
 
 Allows you to view:
-- application icon,
-- package name,
-- name (in all languages),
-- version,
-- build number,
-- the minimum, maximum and target version of the SDK (Android),
-- supported density (DPI) and screen size,
-- architecture (ABI),
-- supported textures,
-- permissions,
-- features,
-- signature,
-- version of OpenGL ES,
-- support Android TV, Wear OS and Android Auto,
-- locales,
+- application icon
+- package name
+- name (in all languages)
+- version
+- build number
+- the minimum, maximum, and target version of the SDK (Android)
+- supported density (DPI) and screen size
+- architecture (ABI)
+- supported textures
+- permissions
+- features
+- signature
+- OpenGL ES version
+- whether app supports Android TV, Wear OS, and/or Android Auto
+- locales
 - a variety of hashes (MD2, MD4, MD5, SHA1, SHA256, SHA384, SHA512)
-- and a lot of other information.
+- and a lot of other information
 
-The application is translated into 86 languages. The language is selected automatically based on the language of the operating system, but you can specify the language in the settings file.
+The application is currently translated into 86 languages.
+The language is selected automatically based on the language of the operating system, but you can specify the language in the settings file.
 
-The app allows you to go to Google Play, a custom store and VirusTotal.
+The app allows you to go to Google Play, APKPure, and VirusTotal.
```

Comment:
The meaning is distorted. ApkPure it is only default custom store, It can be any.